### PR TITLE
[CS] Apply phpdoc_annotation_without_dot

### PR DESCRIPTION
**Pull request by keradus**
| Q             | A
| ------------- | ---
| Branch?       | 2.7
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | n/a
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

Rule was manually applied here : https://github.com/symfony/symfony/pull/19198
In same PR, fixer for this rule was requested, later implementation was approved by Symfony in https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2020 .

Rule is converting single sentence to not a sentence, dropping final dot and lowercasing first word.
If there are multiple sentences, it doesn't make any changes.

Status quo is that some annotation are in the middle - having first word uppercased, but no final stop.
Let us fix grammar by finishing applying the rule.

If, for some reason, you don't want to follow that rule that was requested by Symfony, please provide reasoning. If it's to some edge-case bug, simply raise that bug issue. If due to some other reasons, please send a PR to drop it from `@Symfony` ruleset.

Info: I did manually reviewed every single change of this PR.

### CODE REVIEW

File: src/Symfony/Component/Debug/Tests/Fixtures/DeprecatedClass.php [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 /**
  * @deprecated but this is a test
- *             deprecation notice.
+ *             deprecation notice
```
Comment (chalasr) [inline src/Symfony/Component/Debug/Tests/Fixtures/DeprecatedClass.php]: corresponding assertion needs to be updated

Comment (chalasr) [reply src/Symfony/Component/Debug/Tests/Fixtures/DeprecatedClass.php]: https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Debug/Tests/DebugClassLoaderTest.php#L141

Comment (keradus) [reply src/Symfony/Component/Debug/Tests/Fixtures/DeprecatedClass.php]: Thanks

**Comment by chalasr** [comment]
> Status quo is that some annotation are in the middle - having first word uppercased, but no final stop.

Can we fix them (or avoid lcfirst already updated ones)?

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @keradus.

**Comment by fabpot** [comment]
I've just merged 2.7 up to master. I will now re-execute the fixer on each branch to fix all violations introduced since 2.7.

**Comment by keradus** [comment]
> > Status quo is (...).

> Can we fix them (...) ?

this PR does, at least for 2.7 line

**Comment by keradus** [comment]
thanks @fabpot :+1:

**Comment by chalasr** [comment]
@keradus What I mean is that we now have some annotations with first letter uppercased in the middle of a block, that feels inconsistent. IMHO that should be two different rules, or the rule should be named `phpdoc_annotation_no_sentence` and cover both. Right now php-cs-fixer checks against the dot but acts on both dot and first letter.

**Pull request closed without merging**